src/main/java/jmgomez/apipolicy/service/PolicyService.java

```diff
@@ -1,39 +1,58 @@
 package jmgomez.apipolicy.service;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
 import jmgomez.apipolicy.feignClient.PolicyClient;
 import jmgomez.apipolicy.model.Accident;
-import jmgomez.apipolicy.model.dto.PolicyDto;
-import jmgomez.apipolicy.model.dto.PolicyDtoCov;
+import jmgomez.apipolicy.model.Policy;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
+import java.util.List;
+
 @Service
 public class PolicyService implements IPolicyService {
 
     private final PolicyClient policyClient;
 
+    @Autowired
+    private ObjectMapper defaultObject;
```

Comment:
???!!!!! Why are you injecting an ObjectMapper? You don't even use it afterwards. Beware if you are doing some copy/paste or using AI... this exercise has been designed to learn so you need to understand every line you write